developer-docs/image_sources.md

```diff
@@ -0,0 +1,67 @@
+This document lists all of the images used by RKE2 and any of its packaged components (Helm charts), along with info on where they are built and what other images they copy content from. If you find an image not on this list, please open a PR.
+
+| Chart | Image | Build Repo | Source Images | Hardened |
+| ----- | ----- | ---------- | ------------- | -------- |
+| CORE | rancher/hardened-kubernetes | rancher/image-build-kubernetes | ‣ ubi7/ubi-minimal<br>‣ rancher/hardened-build-base | TRUE |
+| CORE | rancher/rke2-runtime | rancher/rke2 | ‣ rancher/k3s<br>rancher/hardened-kubernetes<br>‣ rancher/hardened-containerd<br>‣ rancher/hardened-crictl<br>‣ rancher/hardened-runc | TRUE |
+| CORE FROM | rancher/hardened-containerd | rancher/image-build-containerd | ‣ ubi7/ubi-minimal<br>‣ rancher/hardened-build-base | TRUE |
```

Comment:
Can you explain what CORE vs CORE FROM means?

Reply:
done